distribution/test/itests/pom.xml

```diff
@@ -34,7 +34,7 @@
         <pax.tinybundles.version>2.1.1</pax.tinybundles.version>
         <pax.url.version>2.4.5</pax.url.version>
         <!-- End pax exam properties -->
-        <cometd.version>3.0.9</cometd.version>
+        <cometd.version>3.1.4</cometd.version>
```

Comment:
❓ Shouldn't we move that up to the main pom? It seems to be use in a couple of places

Reply:
Probably, but since this is unlikely to need to change on the release branch ever again, I'm going to leave it for now.